builder

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env perl
 
 use Getopt::Long;
+use List::MoreUtils qw(uniq);
```

Comment:
Should have checked for that first; removed the use and falling back to the local subroutine.